Added Release Disclaimers

klumb opened the pull request.

klumb commented:
@GeorgLink sorry, this is a little messy.

I had a merge conflict and a previous PR hadn't been sent through. Not sure why DCO is failing for the last two but It is grabbing my DCO for the first so I am setting DCO to pass.

GeorgLink commented:
There are a lot of changes including file name changes. Would you mind creating a new (cleaner) PR from the current master branch?

klumb commented:
@GeorgLink I will just cancel this PR and let someone else do the work.

germonprez commented:
Could you tell me what the work was with this PR? I will do it.

GeorgLink commented:
The PR was for the release disclaimers at the top of "under-review" metrics. For example:

> ### This metric is a release candidate. To comment on this metric please see [Issue #127](https://github.com/chaoss/wg-value/issues/127). Following a comment period, this metric will be included in the next regular release.

klumb commented:
- Added the release disclaimer to project-popularity.md
- Removed text from project-popularity.md - "spinning the popularity wheel." and the space between description sentences
- Added the release disclaimer to social listening
- Renamed the metrics template document from _metrics/project-popularity.md_ to _metrics/template.md_ (the project-popularity.md file exists in the focus-areas/communal-value/ folder)

germonprez commented:
I'm not sure I totally understand this last bullet point @klumb

klumb commented:
@germonprez 
For some reason, this file exists:
https://github.com/chaoss/wg-value/blob/master/metrics/project_popularity.md

I found it during the release for the project popularity metric (which is located elsewhere). 

I am not sure why it exists but it does make sense as a metrics markdown template file, so I changed the name to template.md

The alternative is to delete it.

germonprez commented:
Got it. I propose delete. We don't need random templates around.

matt


On Wed, Feb 3, 2021 at 4:23 PM Kevin Lumbard <notifications@github.com>
wrote:

> @germonprez <https://github.com/germonprez>
> For some reason, this file exists:
>
> https://github.com/chaoss/wg-value/blob/master/metrics/project_popularity.md
>
> I found it during the release for the project popularity metric (which is
> located elsewhere).
>
> I am not sure why it exists but it does make sense as a metrics markdown
> template file, so I changed the name to template.md
>
> The alternative is to delete it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/chaoss/wg-value/pull/130#issuecomment-772867642>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFAGUE6XCS77THZWXKD4J3S5HEHTANCNFSM4XBHIXGQ>
> .
>


-- 
Mutual of Omaha Professor
Information Systems
College of Information Science & Technology
University of Nebraska Omaha
CV: https://bit.ly/2kftONr

Vice President
Omaha Parks Foundation

CHAOSS Board Member
https://chaoss.community/
matt@chaoss.community

he / him / his
